src/pages/Authentication/Login/Login.jsx

```diff
@@ -1,19 +1,78 @@
 import { Link } from "react-router-dom";
+import { useState } from "react";
 import {
 	InputTypeOne,
 	InputTypeThree,
 	InputTypeTwo,
 } from "../../../components";
+import { useUser, useAuth, useLoader } from "../../../contexts";
 import { useDocumentTitle } from "../../../customHooks";
 import "./Login.css";
+import { signIn } from "../../../services";
 
 export const Login = () => {
 	useDocumentTitle("Login | MS");
+	const { loginData, dispatch, initialFormState } = useUser();
+	const { toggleAuth } = useAuth();
+	const [rememberMe, setRememberMe] = useState(false);
+	const [isVisible, setIsVisible] = useState(false);
+	const [show, setShow] = useState(false);
+	const [statusMsg, setStatusMsg] = useState("");
+	const { toggleLoader } = useLoader();
+
+	const toggleShow = () => {
+		setShow((e) => !e);
+	};
+
+	const toggleRememberMe = () => {
+		setRememberMe((rememberMe) => !rememberMe);
+	};
+	const toggleVisibility = () => {
+		setIsVisible((visible) => !visible);
+	};
+
+	const onChangeHandler = (e) => {
+		dispatch({
+			type: "HANDLE_LOGIN_INPUT",
+			field: e.target.name,
+			payload: e.target.value,
+		});
+	};
+	const onSubmitHandler = (e) => {
+		e.preventDefault();
+		signIn({
+			loginData,
+			dispatch,
+			initialFormState,
+			toggleAuth,
+			toggleShow,
+			setStatusMsg,
+			rememberMe,
+			toggleLoader,
+		});
+	};
+
+	// const statusMsg = (() => {
+	// 	error ? "toggleError"
+	// })();
 
 	return (
 		<main>
 			<div className="center">
-				<form className="form flex" method="get">
+				{show ? (
+					<div className="alerts flex-col-align-center fw">
+						<div className="alert alert--danger alert--dismissable">
+							{statusMsg}
+							<button
+								onClick={() => setShow((s) => !s)}
+								className="btn btn--icon btn--close--transparent alert--btn__dismiss btn--circular"
+							>
+								<i className="bx bx-x"></i>
+							</button>
+						</div>
+					</div>
+				) : null}
```

Comment:
avoid using null here you could have used ""

Reply:
I don't want to render anything on false condition so i have used null instead of string which can render in JSX.